src/Mappers/ElementMapper.cs

```diff
@@ -204,20 +209,19 @@ private object GetFileUploadElementValue(string key, FormAnswers formAnswers)
             var value = formAnswers.Pages.SelectMany(_ => _.Answers)
                 .Where(_ => _.QuestionId.Equals(appointmentId) || _.QuestionId.Equals(appointmentDate) ||
                             _.QuestionId.Equals(appointmentStartTime) || _.QuestionId.Equals(appointmentEndTime) ||
-                            _.QuestionId.Equals(appointmentLocation) || _.QuestionId.Equals(appointmentHashedId))
+                            _.QuestionId.Equals(appointmentLocation))
                 .ToList();
 
             if (!value.Any())
                 return null;
 
             var bookingId = value.FirstOrDefault(_ => _.QuestionId.Equals(appointmentId))?.Response;
-            var bookingHashedId = value.FirstOrDefault(_ => _.QuestionId.Equals(appointmentHashedId))?.Response;
             var bookingDate = value.FirstOrDefault(_ => _.QuestionId.Equals(appointmentDate))?.Response;
             var bookingStartTime = value.FirstOrDefault(_ => _.QuestionId.Equals(appointmentStartTime))?.Response;
             var bookingEndTime = value.FirstOrDefault(_ => _.QuestionId.Equals(appointmentEndTime))?.Response;
             var bookingLocation = value.FirstOrDefault(_ => _.QuestionId.Equals(appointmentLocation))?.Response;
             bookingObject.Id = bookingId != null ? Guid.Parse(bookingId) : Guid.Empty;
-            bookingObject.HashedId = bookingHashedId;
+            bookingObject.HashedId = _hashUtil.Hash(bookingObject.Id.ToString());
```

Comment:
we could hash an empty string, do want to do that? check if id is not null or empty frist then hash if needed.